test/functional/rpc_net.py

```diff
@@ -57,14 +57,14 @@ def run_test(self):
         self.connect_nodes(0, 1)
         self.sync_all()
 
-        self.test_connection_count()
-        self.test_getpeerinfo()
-        self.test_getnettotals()
-        self.test_getnetworkinfo()
+        # self.test_connection_count()
+        # self.test_getpeerinfo()
+        # self.test_getnettotals()
+        # self.test_getnetworkinfo()
         self.test_getaddednodeinfo()
-        self.test_service_flags()
-        self.test_getnodeaddresses()
-        self.test_addpeeraddress()
+        # self.test_service_flags()
+        # self.test_getnodeaddresses()
+        # self.test_addpeeraddress()
```

Comment:
might want to uncomment these other tests

Reply:
Ups, my bad